audb/core/dependencies.py

```diff
@@ -524,10 +525,27 @@ def _column_loc(
         files: typing.Union[str, typing.Sequence[str]],
         dtype: typing.Callable = None,
     ) -> typing.Union[typing.Any, typing.List[typing.Any]]:
-        r"""Column content for selected files."""
-        value = self._df.at[files, column]
+        r"""Column content for selected files.
+
+        Args:
+        column: one of the names in self._schema
+        files: rows to query, index is a filename
+        dtype: convert data to dtype
+
+        Returns: scalar value or list
+        """
```

Comment:
```suggestion
        Args:
            column: one of the names in ``Dependencies._schema``
            files: rows to query, index is a filename
            dtype: convert data to dtype

        Returns:
            scalar value or list

        """
```

Reply:
I am commiting the suggestion in order to resolve conversation. At the end this will change again as `files` input will not be needed  - as it deteriorates performance.